pkg/alicloud/secret.go

```diff
@@ -35,22 +36,32 @@ const (
 	AccessKeyID = "accessKeyID"
 	// AccessKeySecret is the data field in a secret where the access key secret is stored at.
 	AccessKeySecret = "accessKeySecret"
+
+	// DNSAccessKeyID is the data field in a DNS secret where the access key id is stored at.
+	DNSAccessKeyID = "ACCESS_KEY_ID"
```

Comment:
I would suggest to use dnsAccessKeyID / dnsAccessKeySecret.

Reply:
Done